system/CLI/CLI.php

```diff
@@ -437,7 +437,7 @@ public static function newLine(int $num = 1)
 		// Do it once or more, write with empty string gives us a new line
 		for ($i = 0; $i < $num; $i ++)
 		{
-			static::write('');
+			static::write();
```

Comment:
This needs the empty string left in order to get the new line on the CLI.

Reply:
the parameter is default to empty string `''`, so it no need a passing an empty string: 

```php
public static function write(string $text = '', string $foreground = null, string $background = null)
```

Reply:
proof https://3v4l.org/VHqUI

Reply:
Oh, you're right. Ignore :)